### PR TITLE
Added 66 new lists and 1 license

### PR DESCRIPTION
**Pull request by DandelionSprout**
I came across https://kinoar.github.io/xlsx-to-json/, which saved me 2 hours of what would otherwise have been very boring work.

So here's a pull request with all the new lists after all, I guess. The outcome of the converter tool is ever so slightly wonky, but still. This pull request covers #95, #224, #273, #299, #306, #319, #358 and #364, and partially covers #99 and #176.

### CODE REVIEW

File: data/FilterList.json [hunk 1]
```diff
@@ -3574,12 +3574,12 @@
     "publishedDate": null,
     "submissionUrl": null,
     "syntaxId": null,
-    "viewUrl": "http://rlwpx.free.fr/WPFF/hrsk.7z"
+    "viewUrl": "https://v.firebog.net/hosts/Airelle-hrsk.txt"
```
Comment (collinbarrett) [inline data/FilterList.json]: nice! note to self: https://github.com/collinbarrett/FilterLists/issues/201#issuecomment-415001371

Comment (collinbarrett) [reply data/FilterList.json]: ...too bad there isn't one for id 185. I don't seem to see it.

**Comment by collinbarrett** [comment]
Note: just reverted using mirrors for some `.7z` since we can now crawl the .7z originals. Having a 1-click over to a plain-text mirror seems useful to the user, but, for now, I'd really like for FilterLists to avoid relying on or redirecting to mirrors (preferring original sources), since we can't properly vet a mirror host.

https://github.com/collinbarrett/FilterLists/commit/16f918616b8ccaacfd3a798982105a194dfa0525

**Comment by collinbarrett** [comment]
Hey, @DandelionSprout , do you know if this PR covered all of the lists for #95 / #403 ? Or were there more that you didn't capture (totally fine if so, just deciding if we need to keep an issue open or not)?

**Comment by DandelionSprout** [comment]
It's hard to be completely sure about which of those that I've added or not, as Firebog's use of URLs without titles makes it hard to compare with FilterLists' lists.

But it seems that I ended up skipping eight-ish of their lists:

• https://www.dshield.org/feeds/suspiciousdomains_Low.txt
• https://www.dshield.org/feeds/suspiciousdomains_Medium.txt
• https://www.dshield.org/feeds/suspiciousdomains_High.txt
• https://raw.githubusercontent.com/CHEF-KOCH/Canvas-Font-Fingerprinting-pages/master/Canvas.txt
• https://raw.githubusercontent.com/CHEF-KOCH/Audio-fingerprint-pages/master/AudioFp.txt
• https://raw.githubusercontent.com/CHEF-KOCH/Canvas-fingerprinting-pages/master/Canvas.txt
• https://raw.githubusercontent.com/chadmayfield/my-pihole-blocklists/master/lists/pi_blocklist_porn_all.list
• https://raw.githubusercontent.com/chadmayfield/my-pihole-blocklists/master/lists/pi_blocklist_porn_top1m.list

**Comment by collinbarrett** [comment]
Right, it's a lot of tedious work... Ok, thanks! I copied those over to #403 to review if we still want to add them.

**Pull request merged**